egs2/misp2021/avsr1/extractor/LICENSE

```diff
@@ -0,0 +1,33 @@
+License
```

Comment:
Did you get these files from some repo?
If so, 
1. please add such information
2. how about including `git clone` of this repo in the data preparation stage instead of including the copy of them?